src/tests/ftest/deployment/io_sys_admin.yaml

```diff
@@ -67,13 +67,13 @@ largefilecount:
     - HDF5-VOL
     - DFS
   object_class:
-    - SX
-    - EC_2P1GX
+    - [SX, EC_2P1GX]
+    - [S1, EC_2P1G1]
```

Comment:
Can we please split this into ior_object_class and mdtest_object_class so the config and logic is more clear?
And could we leave a comment saying something like `file oclass, dir oclass`?
Also, for mdtest we can use `*X` for the _**directory**_

Reply:
Also, do we really intend to use SX/S1 for files and EC for directories?

Reply:
Or is this not file, dir? I think splitting into ior_object_class and mdtest_object_class will make this easier to understand